content/en/agent/amazon_ecs/tags.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Amazon ECS Tag Extraction
+kind: documentation
+further_reading:
+- link: "/getting_started/tagging/"
+  tag: "Documentation"
+  text: "Getting started with tags"
+- link: "/getting_started/tagging/using_tags/"
+  tag: "Documentation"
+  text: "Using tags with Datadog"
+- link: "/agent/guide/autodiscovery-management/"
+  tag: "Documentation"
+  text: "Limit data collection to a subset of containers only"
+---
+
+## Overview
+
+The Datadog Agent can create and assign tags to all metrics, traces, and logs emitted by a container based on its labels or environment variables.
+
+## Unified service tagging
+
+As a best practice in containerized environments, Datadog recommends using unified service tagging when assigning tags. Unified service tagging ties Datadog telemetry together through the use of three standard tags: `env`, `service`, and `version`. To learn how to configure your environment with unified tagging, refer to the dedicated [Amazon ECS unified service tagging documentation][1].
+
+## Resource tag collection
+
+If you do not have unified service tagging enabled, complete the following steps to collect ECS resource tags:
+
+1. Verify your [Amazon ECS container instances][2] are associated to an IAM role. This can be done when creating a new cluster with the ECS cluster creation wizard or in the launch configuration used by an autoscaling group.
```

Comment:
```suggestion
1. Verify your [Amazon ECS container instances][2] are associated with an IAM role. This can be done when creating a new cluster with the ECS cluster creation wizard or in the launch configuration used by an autoscaling group.
```